packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -301,13 +328,20 @@ export async function publicOidc(ctx: Ctx<void, GetPublicOIDCConfigResponse>) {
   try {
     // Find the config with the most granular scope based on context
     const config = await configs.getOIDCConfig()
+    const oidcLogoConfig = await configs.getOIDCLogosDoc()
+
+    if (oidcLogoConfig) {
+      enrichOIDCLogos(oidcLogoConfig)
+    }
 
     if (!config) {
       ctx.body = []
     } else {
       ctx.body = [
         {
-          logo: config.logo,
+          logo: oidcLogoConfig
```

Comment:
Can this be simplified? Something like:

```js
logo: oidcLogoConfig?.config[config.logo] ?? config.logo
```

Reply:
Yep I agree with Mel here, definitely should be simplified into a one liner. On a separate note, the naming here is a little confusing haha.